client/my-sites/theme/live-preview-button/index.jsx

```diff
@@ -0,0 +1,129 @@
+import { Button } from '@automattic/components';
+
+/**
+ * Hardcoded list of themes that are not compatible with Block Theme Previews.
+ * This list should be removed once they are retired.
+ *
+ * @see pekYwv-284-p2
+ */
+const NOT_COMPATIBLE_THEMES = [
+	'premium/alonso',
+	'premium/baxter',
+	'premium/byrne',
+	'premium/kerr',
+	'premium/munchies',
+	'premium/payton',
+	'premium/skivers',
+	'premium/thriving-artist',
+	'pub/ames',
+	'pub/antonia',
+	'pub/appleton',
+	'pub/arbutus',
+	'pub/attar',
+	'pub/barnett',
+	'pub/bennett',
+	'pub/calvin',
+	'pub/calyx',
+	'pub/course',
+	'pub/dorna',
+	'pub/farrow',
+	'pub/geologist-blue',
+	'pub/geologist-cream',
+	'pub/geologist-slate',
+	'pub/geologist-yellow',
+	'pub/hari',
+	'pub/heiwa',
+	'pub/jackson',
+	'pub/kingsley',
+	'pub/marl',
+	'pub/meraki',
+	'pub/quadrat-black',
+	'pub/quadrat-green',
+	'pub/quadrat-red',
+	'pub/quadrat-white',
+	'pub/quadrat-yellow',
+	'pub/quadrat',
+	'pub/russell',
+	'pub/seedlet-blocks',
+	'pub/winkel',
+];
+
+const isNotCompatibleThemes = ( stylesheet ) => {
+	return NOT_COMPATIBLE_THEMES.includes( stylesheet );
+};
+
+/**
+ * Live Preview leveraging Gutenberg's Block Theme Previews
+ *
+ * @see pbxlJb-3Uv-p2
+ */
+export const LivePreviewButton = ( {
+	isActive,
+	isAtomic,
+	isExternallyManagedTheme,
+	isFullSiteEditingTheme,
+	isSimple,
+	isThemeInstalledOnAtomicSite,
+	isWporg,
+	siteSlug,
+	stylesheet,
+	themeType,
+	themeId,
+	translate,
+} ) => {
+	// A user doesn't want to preview the active theme.
+	if ( isActive ) {
+		return null;
+	}
+
+	// A theme should be FullSiteEditing compatible to use Block Theme Previews.
+	if ( ! isFullSiteEditingTheme ) {
+		return null;
+	}
+
+	/**
+	 * Block Theme Previews do not support themes with a static page as a homepage
+	 * as the Site Editor cannot control Reading Settings.
+	 *
+	 * @see pekYwv-284-p2#background
+	 */
+	if ( isNotCompatibleThemes( stylesheet ) ) {
```

Comment:
Do we need to account for Atomic site here? Or just Simple sites? Asking because the theme slugs on Atomic sites probably won't be prefixed with `/pub` or `/premium` CMIIW

Reply:
`stylesheet` includes prefixes (`pub/`, `premium/`) even on Atomic sites!

This is the reason I switch `themePath` depending on if it's Atomic or Simple. here https://github.com/Automattic/wp-calypso/pull/79775/files/ac6c6b80c7ed9a71ab8d736db5254a1e8ce4eceb#diff-6535b936322e5679c10cdf1adeac00cda6c355312582892d7a634c4e2d6fbd64R120-R124